src/index.js

```diff
@@ -0,0 +1,29 @@
+import './style.css';
+
+const itemsList = [{
+  index: 0,
+  description: 'Finish the to-do list',
+  completed: false,
+},
+{
+  index: 1,
+  description: 'Clean the house',
+  completed: false,
+},
+{
+  index: 2,
+  description: 'wash the dishes',
+  completed: true,
+},
+];
+
+const toDoListContainer = document.querySelector('.to-do-list-container');
+
+const itemsListDisplay = () => {
+  itemsList.forEach((item) => {
+    toDoListContainer.innerHTML += `
+    <li class='to-do-item' value='index'>${item.description}<i class="fa-solid fa-ellipsis-vertical"></i></li>`;
+  });
+};
```

Comment:
- [ ] Each task should have a checkbox that we can toggle as completed or incomplete based on the design. Kindly stick to the design in this [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg) by making every task have a checkbox. 👍 😎